src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.Windows.cs

```diff
@@ -37,5 +37,19 @@ public bool LoadUserProfile
             get { return _loadUserProfile; }
             set { _loadUserProfile = value; }
         }
+
+        // CoreCLR can't correctly support UseShellExecute=true for the following reasons
+        // 1. ShellExecuteEx is not supported on onecore.
```

Comment:
WACK compliance is not an issue here, it is only an issue for store apps and Process is not supported on store. The light-up in such scenarios is challenging since there is no deterministic failure or gracefully exit (at least that was the last time we checked with windows) that you will get in case you depend on a non-OneCore compliant APIs. ShellExecuteEx usage is particularly challenging, since some shell extensions require to be instantiated in COM STA threads which forces us to switch apartment states, a notion we do not support in NetCore. 

Windows at some point did expose some mechanism to query for availability but for Corefx to consume it, a runtime interop feature will be required and there was never a strong case to do that.